### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by goord** [goord/dales#16]
The DALES python interface needs sphinx documentation and an entry in readthedocs.io. This has high priority because other issues depend on this.

**Comment by fjansson** [comment]
There is now a skeleton for Sphinx documentation set up in`omuse/doc`.  For each model, the documentation can be placed in a 'doc' inside the models directory. For Dales, this is already done.

To build the documentation, run `make html` in the doc directory.  The compiled documentation is placed in  `omuse/doc/_build/`.

If OMUSE was installed in a virtual Python environment, the documentation should be build with the same environment active. Use `pip install sphinx sphinx_rtd_theme` to install Sphinx in the environment.

Still to do:
* extend the Dales documentation. 
    * list the grids and their variables
    * show how grids are accessed asynchronously (once this is merged into AMUSE)
    * describe examples
* add to readthedocs.io
* add links to documentation to the paper metadata table
